src/tests/EndToEndSpec.scala

```diff
@@ -109,12 +109,12 @@ case class EndToEndSpec()(implicit runner: Runner) {
       }
   }
 
-  test("fetch entities - simple")(
-    Dao[TestSimpleEntity].all().toVector.sortBy(_.id)
-  ).assert(_ == (simpleEntities ++ batchedSimpleEntities).sortBy(_.id))
-  test("fetch entities - complex - long id")(
-    Dao[TestComplexLongId].all().toVector.sortBy(_.id)
-  ).assert(_ == longIdComplexEntities.sortBy(_.id))
+  // test("fetch entities - simple")(
+  //   Dao[TestSimpleEntity].all().toVector.sortBy(_.id)
+  // ).assert(_ == (simpleEntities ++ batchedSimpleEntities).sortBy(_.id))
+  // test("fetch entities - complex - long id")(
+  //   Dao[TestComplexLongId].all().toVector.sortBy(_.id)
+  // ).assert(_ == longIdComplexEntities.sortBy(_.id))
```

Comment:
Is this work in progress?

Reply:
Ohh, I've must commented it out while catching a bug. Fixed